### ISSUE TITLE
Method or way to destroy calendar object?

### ISSUE THREAD

**Issue by jberlet** [EddyVerbruggen/Calendar-PhoneGap-Plugin#171]
I am currently creating my calendar object within an init() function inside a "deviceready" block. Calendar object initializes without issue however I am invoking this calendar object (using createEventInteractively) in another function that is also within the same "deviceready" block. 

I am getting multiple calendar objects left in memory/running as I traverse my app. (using another plugin within app. etc.). I can see there are multiple calendar objects open as when I press 'Cancel' within Calendar activity/widget, it closes a top calendar window to reveal another calendar widget/activity.

Is there a way to destroy the calendar object explicitly with a calendar.remove() method etc. or other means?

Thanks!
JB

**Comment by EddyVerbruggen** [comment]
Let's get this straight before saying something silly: you are creating the JavaScript object `var calendar = ...`. I get that.

When you say: "I am getting multiple calendar objects left in memory/running", where do you see those?

Also, in what way does this bother you exactly?

**Comment by jberlet** [comment]
Eddy - thanks, I ended up not using the CreateInteractive method - too much trouble to deal with user interaction and no way to distinguish between Add and Cancel return codes besides the method you suggested(findEvent in the Success callback - which would require creating unique-identified calendar events). This issue was caused by the way my app. was launching the calendar and not the initialization method.

This issue can be closed.
Thanks.
JB
